### PR TITLE
[#11861] Instructor enrolling students: Fix scroll down link for instructor enroll student page

### LINKED ISSUE #11861: Instructor enrolling students: 'scroll down' link broken
v8.16.1

Steps:
1.  Login as an instructor
2.  go to the page to enroll students into a course.
3. click on the following link

image

problem: Nothing happens
expected: Should scroll to the info referred to in the text. 

However, there is no such info in the page. Not sure if the removal of that info was intended in which case the link should (the entire sentence) should be removed. Otherwise the missing info need to be added back.

### PR DESCRIPTION
**Pull request by sivayogasubramanian**
Fixes #11861

**Outline of Solution**

After enrolling students, the instructor is still able to enroll more students. Therefore, it makes sense to still have the information about how to use the spreadsheet on the page even after enrolling some students. 

https://user-images.githubusercontent.com/53614454/175281529-4cbfcd41-5cd0-4e8d-b612-44bb0eef2c36.mp4

### CODE REVIEW

**Comment by sivayogasubramanian** [comment]
@damithc Any feedback?

**Comment by damithc** [comment]
> @damithc Any feedback?
@sivayogasubramanian I'm OK with the UI change. Let's wait for the dev team to review the code.

**Review by zhaojj2209 — APPROVED** [review]
LGTM!

However, I've noticed the "Enroll Students" button displays weirdly after clicking; I have raised a separate issue #11876 for it since it is not related to this PR.

File: src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.html [hunk 1]
```diff
@@ -122,7 +122,45 @@ <h1 id="enroll-header" class="text-break">Enroll Students for {{ courseId }}</h1
...
+    <div id="results-panel" *ngIf="coursePresent && showEnrollResults">
+      <h1 class="text-break">Enrollment Results for {{ courseId }}</h1>
+      <div *ngFor="let enrollResultPanel of enrollResultPanelList">
+        <div class="card enroll-results-panel" *ngIf="enrollResultPanel.studentList.length > 0">
+          <div class="card-header" [ngClass]="{'bg-success': enrollResultPanel.status === EnrollStatus.NEW,
+       'bg-warning': enrollResultPanel.status === EnrollStatus.MODIFIED,
+       'bg-secondary': enrollResultPanel.status === EnrollStatus.MODIFIED_UNCHANGED,
+       'bg-danger': enrollResultPanel.status === EnrollStatus.ERROR,
+       'bg-default': enrollResultPanel.status === EnrollStatus.UNMODIFIED}" [style.color]="enrollResultPanel.status ===
+       EnrollStatus.NEW || enrollResultPanel.status === EnrollStatus.MODIFIED_UNCHANGED ? 'white' : 'black' ">
```
Comment (samuelfangjw) [inline src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.html:134]: nit: indentation of these lines seems a little off, can refer to how they were indented originally. Rest of the PR looks good.

Comment (samuelfangjw) [reply src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.html]: `tr` elements need an extra level of indent as well.

Comment (sivayogasubramanian) [reply src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.html]: Fixed in https://github.com/TEAMMATES/teammates/pull/11872/commits/c260361a2f9298f58367d5b830f8744542255646

**Review by samuelfangjw — APPROVED** [review]

**Pull request merged**